From c34d203b05ea3e529eef81f204a67b0537576cec Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Sun, 26 Feb 2017 12:36:45 +0200
Subject: [PATCH 0810/1345] fix: dts: a8040-mcbin: Enable COMPHY devices in DT

commit  963559bf4cb6ae56317b386dce3cc009f0dcbff2 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Each SATA device node includes a pointer (phandle) to a phy unit
(COMPHY), but the COMPHY node itself is not enabled and as result the
COMPHY driver is not loaded.
It leads to a SATA device detection failure.
This patch enables COMPHY nodes in Macchiatobin DTS file.

Change-Id: Ib8c9b3869b2972063ca8a9db661531cf49df356b
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37002
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37107
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index b29972a..9bbea04 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -81,6 +81,9 @@
 					marvell,function = "gpio";
 				};
 			};
+			comphy0: comphy {
+				status = "okay";
+			};
 
 			gop {
 				emac0_0: mac0 {
@@ -113,6 +116,9 @@
 
 	cp110-slave {
 		config-space@f4000000 {
+			comphy1: comphy {
+				status = "okay";
+			};
 			gop {
 				emac0_1: mac0 {
 					phy-mode = "sfi"; /* lane-4 */
-- 
1.7.9.5

